sinks/kafka/kafka_test.go

```diff
@@ -231,11 +236,116 @@ func TestSpanConstructor(t *testing.T) {
 	assert.Equal(t, time.Second*10, sink.config.Producer.Flush.Frequency, "flush frequency did not set correctly")
 }
 
+func TestSpanSamplingTooHigh(t *testing.T) {
```

Comment:
Add one of these for the too-low case too? (Either verifying a new value or that it errors as expected depending on your choice from my previous comment.)

Reply:
I added a test for the too-low case in the `TestSpanInstantiateFailure` test.